src/apis/homegraph/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@google/homegraph",
+  "version": "0.1.0",
+  "description": "homegraph",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "keywords": [
+    "google"
+  ],
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/googleapis/google-api-nodejs-client",
+  "bugs": {
+    "url": "https://github.com/googleapis/google-api-nodejs-client/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googleapis/google-api-nodejs-client.git"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "fix": "gts fix",
+    "lint": "gts check",
+    "compile": "tsc -p .",
+    "prepare": "npm run compile",
+    "docs": "typedoc --out docs/",
+    "webpack": "webpack"
+  },
+  "dependencies": {
+    "googleapis-common": "^2.0.4"
+  },
+  "devDependencies": {
+    "gts": "^1.0.0",
+    "null-loader": "^3.0.0",
+    "ts-loader": "^6.0.4",
```

Comment:
I don't believe that the generator generates with up-to-date dependencies, so you might want to update these accordingly.